scss/_card.scss

```diff
@@ -91,6 +91,10 @@
 // Background variations
 //
 
+.card-default {
+  background-color: #fff;
```

Comment:
Was going to use `$gray-lightest` but it wasn't the same in v3. I can create another commit adding a variable, or I could use `$body-bg` if you'd like.